cmd/scollector/main.go

```diff
@@ -99,6 +99,7 @@ func main() {
 		}
 	}
 	collectors.Init(conf)
+	check(collectors.AddIfstatConfig(conf.IfaceExpr))
```

Comment:
We are going away from having main initialize collectors. Instead conllectors can "self-register" init functions. See https://github.com/bosun-monitor/bosun/blob/master/cmd/scollector/collectors/haproxy_unix.go#L16